### PR TITLE
TouchBlick fix after reformat code

### PR DESCRIPTION
**Pull request by SCratORS**

### CODE REVIEW

File: java/com/scrat/everchanging/EverchangingRender.java [hunk 1]
```diff
@@ -200,11 +199,12 @@ public void onSurfaceChanged(final GL10 gl, final int width, final int height) {
         GLES20.glViewport(0, 0, width, height);
 
         final float scaleImageHeight = surfaceHeight / 320f;
+        final float scaleImageWidth = 240f / width;
         //todo setup position
         final int scenesSize = scenes.size();
         for (int i = 0; i < scenesSize; i++) {
             scenes.get(i)
-                    .setupPosition(surfaceWidth, surfaceHeight, scaleImageHeight, displayRotation);
+                    .setupPosition(surfaceWidth, surfaceHeight, (i<scenesSize-1)?scaleImageHeight:scaleImageWidth, displayRotation);
```
Comment (Doctoror) [inline java/com/scrat/everchanging/EverchangingRender.java:207]: Sorry if I broke this. This code is not formatted now BTW (no spaces between operators).
Usually when I make any changes I press IDE shortcut to run a formatter.
Please consider doing the same for consistent formatting.

Comment (Doctoror) [reply java/com/scrat/everchanging/EverchangingRender.java:207]: See https://trunk.io/learn/what-is-a-code-formatter-and-why-would-i-want-to-use-one

Comment (SCratORS) [reply java/com/scrat/everchanging/EverchangingRender.java:207]: I'm not against it, but I haven't learned how to use it yet.

Comment (Doctoror) [reply java/com/scrat/everchanging/EverchangingRender.java:207]: If you are using Android Studio on Linux, the default shortcut might be `Ctrl + Alt + L`.
You can find it here
Format code

And you can change the shortcut here
[
Formatter shortcut
](url)

I also recommend running "Optimize imports". It removes unused imports and rearranges them alphabetically.
Optimize imports

Comment (SCratORS) [reply java/com/scrat/everchanging/EverchangingRender.java:207]: I use Android Studio on Windows. Yes, it's in the same place. I will use it. Thanks for the hint

**Pull request merged**